a10_octavia/db/repositories.py

```diff
@@ -183,9 +183,10 @@ def get_all_deleted_expiring(self, session, exp_age):
 class VThunderRepository(BaseRepository):
     model_class = models.VThunder
 
-    def get_recently_updated_thunders(self, session, expiry_time):
+    def get_recently_updated_thunders(self, session):
         query = session.query(self.model_class).filter(
-            self.model_class.updated_at >= expiry_time).filter(
+            or_(self.model_class.updated_at >= self.model_class.last_write_mem,
+                self.model_class.last_write_mem == None)).filter(
             self.model_class.status == 'ACTIVE')
```

Comment:
You can add a status as `DELETED` here, to take care of write memory on `vthunders` records, on which the last LB got deleted in cases mentioned in https://a10networks.atlassian.net/browse/STACK-2028

Reply:
done